.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Report an issue with DXC
+title: ''
+labels: ['bug']
```

Comment:
shall we add a "needs triage" label by default on all templates and clear them if one of us have interacted with it?